### ISSUE TITLE
unable to start project

### ISSUE THREAD

**Issue by tarim** [coryhouse/ps-redux#1]
Hi Cory,

I try to run your project on my local server,and it failed. 

Here is debug result. Please give me some advice. Thanks, 

verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start' ]
2 info using npm@2.14.7
3 info using node@v4.2.2
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info prestart ps-redux@1.0.0
6 verbose unsafe-perm in lifecycle true
7 info ps-redux@1.0.0 Failed to exec prestart script
8 verbose stack Error: ps-redux@1.0.0 prestart: `npm-run-all --parallel start-message remove-dist lint:tools`
8 verbose stack Exit status 1
8 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:214:16)
8 verbose stack     at emitTwo (events.js:87:13)
8 verbose stack     at EventEmitter.emit (events.js:172:7)
8 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:24:14)
8 verbose stack     at emitTwo (events.js:87:13)
8 verbose stack     at ChildProcess.emit (events.js:172:7)
8 verbose stack     at maybeClose (internal/child_process.js:818:16)
8 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
9 verbose pkgid ps-redux@1.0.0
10 verbose cwd C:\Nur\tech\redux\ps-redux
11 error Windows_NT 6.1.7601
12 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
13 error node v4.2.2
14 error npm  v2.14.7
15 error code ELIFECYCLE
16 error ps-redux@1.0.0 prestart: `npm-run-all --parallel start-message remove-dist lint:tools`
16 error Exit status 1
17 error Failed at the ps-redux@1.0.0 prestart script 'npm-run-all --parallel start-message remove-dist lint:tools'.
17 error This is most likely a problem with the ps-redux package,
17 error not with npm itself.
17 error Tell the author that this fails on your system:
17 error     npm-run-all --parallel start-message remove-dist lint:tools
17 error You can get their info via:
17 error     npm owner ls ps-redux
17 error There is likely additional logging output above.
18 verbose exit [ 1, true ]

**Comment by coryhouse** [comment]
This project is under active development. How'd you even find it? That said, please get latest and if it doesn't run, what OS are you on?

**Comment by tarim** [comment]
Hi Cory,

I found it from your reply to bootstrap issue on other project. I did clone it few hours ago, and my environment is windows 7 64 bit computer. Thank you.

**Comment by coryhouse** [comment]
Ah, I forgot I shared the link there. :) Does it run if you get latest? Also, are you using the native windows command line?

**Comment by tarim** [comment]
No, it doesn't , and I am using node js command line.

Thanks,

**Comment by coryhouse** [comment]
I'm confused. You can't run an app in the Node repl. If you try, it'll complain: 
![image](https://cloud.githubusercontent.com/assets/1688997/14505256/0088c66c-017e-11e6-8dda-504a73329eb2.png)

Am I misunderstanding? Does the app run when you use the native Windows command line?

**Comment by tarim** [comment]
No, I am using node command line, I did npm install, then I did npm start -s.

Thanks,

Nur Suleiman

**Comment by coryhouse** [comment]
What is "the node command line"? How are you opening this command line? The only Node command line I'm aware of is the one I showed in the screenshot. (Thanks for your patience!)

**Comment by coryhouse** [comment]
A screenshot would really help :)

**Comment by vfabregat** [comment]
For future references, I had the same error. The problem was that I already used the port 3000 (Wich is the default in the srcServer.js script)

regards.
